app/assets/javascripts/app/discussions/index/index.html.slim

```diff
@@ -23,7 +23,7 @@ div ui-view="sidebar"
         section.CardList.CardList--withAnimation ng-show="closedDiscussions.length"
           .CardList-label Closed Discussions
           .CardList-card ng-repeat="discussion in closedDiscussions = (ctrl.discussions | filter:{open: false})"
-            discussion-card discussion="discussion"
+            discussion-card.Discussion--closed discussion="discussion"
```

Comment:
Fixed.

I also renamed Detailed discussions to Discussions and Discussions to Discussion Cards, as @joaojusto suggested. I believe it is now according to the JS naming as well.